README.md

```diff
@@ -17,9 +17,11 @@ This site is built on top of the critical work done by [@aepyornis](https://gith
 
 Backend logic and data manipulation is largely handled by making calls to PostgreSQL functions and prebuilding results into tables whenever possible to avoid complex queries made per-request. See the [hpd-registration ](https://github.com/nycdb/nycdb/tree/master/src/nycdb/sql/hpd_registrations) scripts of `nycdb` for the SQL code that provides this functionality.
 
+Note that both the backend and the frontend of the app each contain __separate `package.json` configurations__ as well as __separate sets of environment variables__. 
+
```

Comment:
It might be useful to mention that this is the recommended way of doing things according to Create React App, to help convince the reader that we're not just being weird. 😉

Reply:
haha good point